### PR TITLE
Run automated code cleanup

### PR DESCRIPTION
**Pull request by dimhotepus**
* Format sources.
* Remove unused usings.
* Drop unneeded casts.
* Inline out vars.
* Use compound assignments.
* new() / default.
* Add access modifiers.
* Use readonly.
* Use pattern matching.
* Simplify boolean expressions.

Also use correct naming in async / sync test runs.

### CODE REVIEW

File: UnitTests/Code/TestCard.cs [hunk 1]
```diff
@@ -1,8 +1,7 @@
 ﻿
-using System.Globalization;
 using System;
 using System.Diagnostics;
-using Media.Codec;
+using System.Globalization;
```
Comment (juliusfriedman) [inline UnitTests/Code/TestCard.cs:4]: Is it needed?

**Review by juliusfriedman — COMMENTED** [review]
Lots of changes!

Mostly looks good to me, I will finish up the review ASAP.

Thanks again for your contributions!

File: UnitTests/Code/TestCard.cs (hunk 1 above)
Comment (dimhotepus) [reply UnitTests/Code/TestCard.cs:4]: I've just run Visual Studio automatic code cleanup.

`CultureInfo.InvariantCulture` is in `System.Globalization` namespace.

**Comment by dimhotepus** [comment]
> Lots of changes!
> 
> Mostly looks good to me, I will finish up the review ASAP.
> 
> Thanks again for your contributions!

Those changes (except last one) are created by Microsoft Visual Studio automatic code cleanup profiles.

**Review by juliusfriedman — APPROVED** [review]
Everything went well including the testing, Thanks again for your contributions!

**Pull request merged**